packages/test-helpers/utils.js

```diff
@@ -1,16 +1,31 @@
-module.exports = {
-  makeErrorMappingProxy (target) {
-    return new Proxy(target, {
-      get (target, property) {
-        if (property in target) {
-          return target[property]
-        }
+// Only runs given test block when the condition passes
+const onlyIf = condition => {
+  return testBlock => {
+    if (condition()) {
+      return testBlock()
+    }
+  }
+}
 
-        throw new Error(`Could not find error ${property} in error mapping`)
-      },
-      set () {
-        throw new Error('Unexpected set to error mapping')
+const isValidTxHash = txHash => /^0x([A-Fa-f0-9]{64})$/.test(txHash)
```

Comment:
This is the only one that feels kind of out of place... but I'm also not sure where to put it. Maybe in `lib/`?